<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="4" total="29" passed="25">
  <reporter-output>
  </reporter-output>
  <suite name="Default suite" duration-ms="119628" started-at="2018-07-09T11:21:24Z" finished-at="2018-07-09T11:23:24Z">
    <groups>
    </groups>
    <test name="Default test" duration-ms="119628" started-at="2018-07-09T11:21:24Z" finished-at="2018-07-09T11:23:24Z">
      <class name="Admin_home.TestLoanGrant">
        <test-method status="PASS" signature="browserLunch()[pri:0, instance:Admin_home.TestLoanGrant@cc285f4]" name="browserLunch" duration-ms="6861" started-at="2018-07-09T16:51:24Z" finished-at="2018-07-09T16:51:31Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- browserLunch -->
        <test-method status="PASS" signature="loginDetail()[pri:0, instance:Admin_home.TestLoanGrant@cc285f4]" name="loginDetail" duration-ms="2138" started-at="2018-07-09T16:51:31Z" finished-at="2018-07-09T16:51:33Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- loginDetail -->
        <test-method status="PASS" signature="checkAppDetail()[pri:0, instance:Admin_home.TestLoanGrant@cc285f4]" name="checkAppDetail" duration-ms="932" started-at="2018-07-09T16:51:33Z" finished-at="2018-07-09T16:51:34Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- checkAppDetail -->
        <test-method status="PASS" signature="jumpToMemberProfile()[pri:0, instance:Admin_home.TestLoanGrant@cc285f4]" name="jumpToMemberProfile" duration-ms="10101" started-at="2018-07-09T16:51:34Z" finished-at="2018-07-09T16:51:44Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- jumpToMemberProfile -->
        <test-method status="PASS" signature="ChangePasswordForMember()[pri:0, instance:Admin_home.TestLoanGrant@cc285f4]" name="ChangePasswordForMember" duration-ms="2591" started-at="2018-07-09T16:51:44Z" finished-at="2018-07-09T16:51:47Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- ChangePasswordForMember -->
        <test-method status="PASS" signature="backNevigationForMemberModify()[pri:0, instance:Admin_home.TestLoanGrant@cc285f4]" name="backNevigationForMemberModify" duration-ms="11191" started-at="2018-07-09T16:51:47Z" finished-at="2018-07-09T16:51:58Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- backNevigationForMemberModify -->
        <test-method status="FAIL" signature="modifyMemberProfile()[pri:0, instance:Admin_home.TestLoanGrant@cc285f4]" name="modifyMemberProfile" duration-ms="1898" started-at="2018-07-09T16:51:58Z" finished-at="2018-07-09T16:52:00Z">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [40] but found [86]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [40] but found [86]
	at org.testng.Assert.fail(Assert.java:94)
	at org.testng.Assert.failNotEquals(Assert.java:494)
	at org.testng.Assert.assertEquals(Assert.java:123)
	at org.testng.Assert.assertEquals(Assert.java:370)
	at org.testng.Assert.assertEquals(Assert.java:380)
	at Admin_home.MemberProfileModify.modifyMemberProfile(MemberProfileModify.java:30)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:80)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:714)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:901)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1231)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:334)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:329)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:291)
	at org.testng.SuiteRunner.run(SuiteRunner.java:240)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1198)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1123)
	at org.testng.TestNG.run(TestNG.java:1031)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:132)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:230)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:76)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- modifyMemberProfile -->
        <test-method status="PASS" signature="historyTab()[pri:0, instance:Admin_home.TestLoanGrant@cc285f4]" name="historyTab" duration-ms="68" started-at="2018-07-09T16:52:00Z" finished-at="2018-07-09T16:52:00Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- historyTab -->
        <test-method status="PASS" signature="getHistoryTable()[pri:0, instance:Admin_home.TestLoanGrant@cc285f4]" name="getHistoryTable" duration-ms="2505" started-at="2018-07-09T16:52:00Z" finished-at="2018-07-09T16:52:03Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- getHistoryTable -->
        <test-method status="PASS" signature="adModule()[pri:0, instance:Admin_home.TestLoanGrant@cc285f4]" name="adModule" duration-ms="2042" started-at="2018-07-09T16:52:03Z" finished-at="2018-07-09T16:52:05Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- adModule -->
        <test-method status="PASS" signature="modifyAdvertisment()[pri:0, instance:Admin_home.TestLoanGrant@cc285f4]" name="modifyAdvertisment" duration-ms="16365" started-at="2018-07-09T16:52:05Z" finished-at="2018-07-09T16:52:21Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- modifyAdvertisment -->
        <test-method status="PASS" signature="removeAdvertisment()[pri:0, instance:Admin_home.TestLoanGrant@cc285f4]" name="removeAdvertisment" duration-ms="1265" started-at="2018-07-09T16:52:21Z" finished-at="2018-07-09T16:52:22Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- removeAdvertisment -->
        <test-method status="FAIL" signature="addNewAd()[pri:0, instance:Admin_home.TestLoanGrant@cc285f4]" name="addNewAd" duration-ms="5159" started-at="2018-07-09T16:52:22Z" finished-at="2018-07-09T16:52:28Z">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[the valu of title must take 4 to 25 but it take more then 25 expected [25] but found [45]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: the valu of title must take 4 to 25 but it take more then 25 expected [25] but found [45]
	at org.testng.Assert.fail(Assert.java:94)
	at org.testng.Assert.failNotEquals(Assert.java:494)
	at org.testng.Assert.assertEquals(Assert.java:123)
	at org.testng.Assert.assertEquals(Assert.java:370)
	at Admin_home.AdvertisementsManage.addNewAd(AdvertisementsManage.java:81)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:80)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:714)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:901)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1231)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:334)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:329)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:291)
	at org.testng.SuiteRunner.run(SuiteRunner.java:240)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1198)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1123)
	at org.testng.TestNG.run(TestNG.java:1031)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:132)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:230)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:76)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- addNewAd -->
        <test-method status="PASS" signature="disableExpire()[pri:0, instance:Admin_home.TestLoanGrant@cc285f4]" name="disableExpire" duration-ms="5552" started-at="2018-07-09T16:52:28Z" finished-at="2018-07-09T16:52:33Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- disableExpire -->
        <test-method status="PASS" signature="account()[pri:0, instance:Admin_home.TestLoanGrant@cc285f4]" name="account" duration-ms="14041" started-at="2018-07-09T16:52:33Z" finished-at="2018-07-09T16:52:47Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- account -->
        <test-method status="FAIL" signature="clickPayment()[pri:0, instance:Admin_home.TestLoanGrant@cc285f4]" name="clickPayment" duration-ms="2076" started-at="2018-07-09T16:52:47Z" finished-at="2018-07-09T16:52:49Z">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[ it showing more then 'Debit to member' list  expected [Debit to member] but found [Community to member]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError:  it showing more then 'Debit to member' list  expected [Debit to member] but found [Community to member]
	at org.testng.Assert.fail(Assert.java:94)
	at org.testng.Assert.failNotEquals(Assert.java:494)
	at org.testng.Assert.assertEquals(Assert.java:123)
	at org.testng.Assert.assertEquals(Assert.java:176)
	at Admin_home.AccountModule.clickPayment(AccountModule.java:50)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:80)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:714)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:901)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1231)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:334)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:329)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:291)
	at org.testng.SuiteRunner.run(SuiteRunner.java:240)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1198)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1123)
	at org.testng.TestNG.run(TestNG.java:1031)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:132)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:230)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:76)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- clickPayment -->
        <test-method status="PASS" signature="afterPayType()[pri:0, instance:Admin_home.TestLoanGrant@cc285f4]" name="afterPayType" duration-ms="538" started-at="2018-07-09T16:52:49Z" finished-at="2018-07-09T16:52:50Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- afterPayType -->
        <test-method status="PASS" signature="affterCommentClickToSubmit()[pri:0, instance:Admin_home.TestLoanGrant@cc285f4]" name="affterCommentClickToSubmit" duration-ms="6625" started-at="2018-07-09T16:52:50Z" finished-at="2018-07-09T16:52:56Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- affterCommentClickToSubmit -->
        <test-method status="PASS" signature="accoutInfo()[pri:0, instance:Admin_home.TestLoanGrant@cc285f4]" name="accoutInfo" duration-ms="2343" started-at="2018-07-09T16:52:56Z" finished-at="2018-07-09T16:52:59Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- accoutInfo -->
        <test-method status="PASS" signature="accountName()[pri:0, instance:Admin_home.TestLoanGrant@cc285f4]" name="accountName" duration-ms="2261" started-at="2018-07-09T16:52:59Z" finished-at="2018-07-09T16:53:01Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- accountName -->
        <test-method status="PASS" signature="datePassing()[pri:0, instance:Admin_home.TestLoanGrant@cc285f4]" name="datePassing" duration-ms="2575" started-at="2018-07-09T16:53:01Z" finished-at="2018-07-09T16:53:04Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- datePassing -->
        <test-method status="PASS" signature="submit()[pri:0, instance:Admin_home.TestLoanGrant@cc285f4]" name="submit" duration-ms="3251" started-at="2018-07-09T16:53:04Z" finished-at="2018-07-09T16:53:07Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- submit -->
        <test-method status="PASS" signature="loanPageAfterClickSubmit()[pri:0, instance:Admin_home.TestLoanGrant@cc285f4]" name="loanPageAfterClickSubmit" duration-ms="3368" started-at="2018-07-09T16:53:07Z" finished-at="2018-07-09T16:53:10Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- loanPageAfterClickSubmit -->
        <test-method status="PASS" signature="viewLoan()[pri:0, instance:Admin_home.TestLoanGrant@cc285f4]" name="viewLoan" duration-ms="3470" started-at="2018-07-09T16:53:10Z" finished-at="2018-07-09T16:53:14Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- viewLoan -->
        <test-method status="PASS" signature="loanPage()[pri:0, instance:Admin_home.TestLoanGrant@cc285f4]" name="loanPage" duration-ms="5465" started-at="2018-07-09T16:53:14Z" finished-at="2018-07-09T16:53:19Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- loanPage -->
        <test-method status="FAIL" signature="loanDetail()[pri:0, instance:Admin_home.TestLoanGrant@cc285f4]" name="loanDetail" duration-ms="457" started-at="2018-07-09T16:53:19Z" finished-at="2018-07-09T16:53:20Z">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [10] but found [11]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [10] but found [11]
	at org.testng.Assert.fail(Assert.java:94)
	at org.testng.Assert.failNotEquals(Assert.java:494)
	at org.testng.Assert.assertEquals(Assert.java:123)
	at org.testng.Assert.assertEquals(Assert.java:370)
	at org.testng.Assert.assertEquals(Assert.java:380)
	at Admin_home.TestLoanGrant.loanDetail(TestLoanGrant.java:41)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:80)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:714)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:901)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1231)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:334)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:329)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:291)
	at org.testng.SuiteRunner.run(SuiteRunner.java:240)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1198)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1123)
	at org.testng.TestNG.run(TestNG.java:1031)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:132)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:230)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:76)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- loanDetail -->
        <test-method status="PASS" signature="otherDetailForLoan()[pri:0, instance:Admin_home.TestLoanGrant@cc285f4]" name="otherDetailForLoan" duration-ms="276" started-at="2018-07-09T16:53:20Z" finished-at="2018-07-09T16:53:20Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- otherDetailForLoan -->
        <test-method status="PASS" signature="moreDetail()[pri:0, instance:Admin_home.TestLoanGrant@cc285f4]" name="moreDetail" duration-ms="337" started-at="2018-07-09T16:53:20Z" finished-at="2018-07-09T16:53:20Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- moreDetail -->
        <test-method status="PASS" signature="submitButton()[pri:0, instance:Admin_home.TestLoanGrant@cc285f4]" name="submitButton" duration-ms="3802" started-at="2018-07-09T16:53:20Z" finished-at="2018-07-09T16:53:24Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- submitButton -->
      </class> <!-- Admin_home.TestLoanGrant -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
